guides/manual-installation.md

```diff
@@ -340,364 +304,323 @@ journalctl -fu trento-web
+    > **Note:** When creating the trento-net network, Docker typically assigns a default subnet: `172.17.0.0/16`. Ensure that this subnet matches the one specified in your PostgreSQL configuration file, which can be found at `/var/lib/pgsql/data/pg_hba.conf`. If the subnet of `trento-net` differs from `172.17.0.0/16` then adjust `pg_hba.conf` and restart postgresql.
 
-Verifying the subnet of trento-net:
+1. Verifying the subnet of trento-net:
 
-```bash
-docker network inspect trento-net  --format '{{range .IPAM.Config}}{{.Subnet}}{{end}}'
-```
+    ```bash
+    docker network inspect trento-net  --format '{{range .IPAM.Config}}{{.Subnet}}{{end}}'
+    ```
 
-Expected output:
+    Expected output:
 
-```
-172.17.0.0/16
-```
+    ```bash
+    172.17.0.0/16
+    ```
 
 #### Install Trento on docker
 
-> Note: The environment variables listed here are examples and should be considered as placeholders. Instead of specifying environment variables directly in the docker run command, it is recommended to use an environment variable file. Store your environment variables in a file and use the --env-file option with the docker run command. Find detailed instructions on how to use an environment variable file with Docker on [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env).
-
-##### Create the secret environment variables
-
-```bash
-WANDA_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
-TRENTO_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
-ACCESS_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
-REFRESH_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
-```
-
-##### Install trento-wanda on docker
-
-```bash
-docker run -d --name wanda \
-    -p 4001:4000 \
+The environment variables listed here are examples and should be considered as placeholders. Instead of specifying environment variables directly in the docker run command, it is recommended to use an environment variable file. Store your environment variables in a file and use the --env-file option with the docker run command. Find detailed instructions on how to use an environment variable file with Docker on [official Docker documentation](https://docs.docker.com/engine/reference/commandline/run/#env).
+
+1. Create the secret environment variables:
+
+    ```bash
+    WANDA_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
+    TRENTO_SECRET_KEY_BASE=$(openssl rand -out /dev/stdout 48 | base64)
+    ACCESS_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
+    REFRESH_TOKEN_ENC_SECRET=$(openssl rand -out /dev/stdout 48 | base64)
```

Comment:
I'm confused. When you already recommend in the last paragraph to use an environment file, why are these variables not inserted in the file?

Reply:
The honest answer is that this part was created at different times :)

Normally, it is a good practice to create an env file with secrets, and use it.... but it is easier for the user to just copy and paste the commands when reading the documentation for the first time. 
Every Reader, who is familiar with docker, will know how to adjust this, if you would prefer to change this, we can rework this part.

Reply:
My point was that it looks really odd to recommend something, but then deviate and don't follow your own recommendation. :wink:

As the user also goes through the procedure step by step, I think we should follow this recommendation: create a env file and use that.